app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -52,7 +52,7 @@ def self.raw_connect(server, port, username, password, service = "Service")
     Ovirt.const_get(service).new(params)
   end
 
-  def connect(options = {})
+  def other_connect(options = {})
```

Comment:
Two things.  First, I definitely want all connections to go through on place.  I like the idea of knowing that I can look in one place in the code and learn all the different ways that we have to connect to a provider.

Second, I'm not taking a hard stance that "`connect` calls `raw_connect`" is the pattern to follow all the time.

I was gonna say something about how we already do this with other providers, but I realized I'm wrong...OpenStack's Event Capture connection is handled completely outside of this and wholly owned by the [OpenStack Event Monitor](https://github.com/ManageIQ/manageiq/blob/e3807974ba11382f1dd15fcfc994dcd573a96712/gems/pending/openstack/openstack_event_monitor.rb) which sits **nowhere** near the Provider connection logic.

Still, I don't really care about the details around `connect => raw_connect`.  But, I do think we should try to build connections in one place (even if we're lacking in other areas here).